airbyte-oauth/build.gradle

```diff
@@ -8,4 +8,6 @@ dependencies {
     implementation project(':airbyte-config:persistence')
     implementation project(':airbyte-json-validation')
     testImplementation project(':airbyte-oauth')
+
+    implementation group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.141.59'
```

Comment:
why is this adding selenium dependency even though the tests are not using it?

Reply:
I removed it in a later PR